bindings/test/Makefile

```diff
@@ -0,0 +1,85 @@
+# A sample Makefile for building Google Test and using it in user
```

Comment:
Please add a copyright header and update the text here to say that it's a test for the profiler C++ components rather than a sample. I'd take a pass over all comments in the Makefile to make sure they are useful as part of our Makefile rather than more like a doc for gtest.

Reply:
Done.